src/chttpd/src/chttpd.erl

```diff
@@ -141,7 +144,12 @@ start_link(Name, Options) ->
     end.
 
 get_server_options(Module) ->
-    ServerOptsCfg = config:get(Module, "server_options", "[]"),
+    ServerOptsCfg =
+        case Module of
+            "chttpd" -> config:get(Module,
+                "server_options", ?DEFAULT_SERVER_OPTIONS);
+            _ -> config:get(Module, "server_options", "[]")
```

Comment:
You've commented out [`httpd->server_options = [{sndbuf, 262144}]`](https://github.com/apache/couchdb/pull/3609/files#diff-38acf80292ea264190d7a08646c46d571d7ddcb37a749a58fb79c009030dd115L218). 
Should it be `config:get(Module, "server_options", "[{sndbuf, 262144}]")`. If that's the case we should define one more macro constant.

Reply:
Never mind the other type of Module is [`httpsd`](https://github.com/apache/couchdb/blob/3.x/src/chttpd/src/chttpd.erl#L115)